lib/client/client.wilson.js

```diff
@@ -27,6 +27,9 @@
   var _controllerProvider = null;
   var _provider           = null;
 
+  // Decorate Lodash with conveniences
```

Comment:
yeah... For this instance on the client, its a little different since lodash is required from the application level rather than included with the client wilson code itself.  But for the server side node stuff definitely.